### PR TITLE
Intro screen UI changes

### LINKED ISSUE #71: First screen isn't actionable
## Context ##
The first screen doesn't give the user enough context to understand what this app is.  The list items look like they could be buttons.

## Suggestions ##
- [ ] Simplify the heading.  "The CovidNearMe App" is a weird name for a product.  Can we just say "Welcome to CovidNearMe."?
- [ ] Add a heading that provides context to the feature list:

    "CovidNearMe is a US Centers for Disease Control project to help the public respond to the novel coronavirus (COVID-19).  It allows you to:"
- [ ] Change the button copy to "Get started"

## Screenshot ##
https://photos.google.com/share/AF1QipPNzbQ9YfdnsdI-nSG2WbrB9J5y35_mu5uOORQ4X2pNHiUvaFTdWdgB3X9Y6WxfBw/photo/AF1QipOUHVbyVSTDUL5frFr9CdHq9fJsgj3w0ppwfN2X?key=am52QmZoVTZqOWxxWURVVE9EdkJpYUM3LWxfdGlB

## Smartphone: ##
 - Device: Essential
 - OS: Android 10 QP1A.191005.014
 - App version: 6d80da8

### PR DESCRIPTION
**Pull request by aadarsh-patel**
Closes #71 

Hey @joshua-s , 
Requesting for a code review.
And I also have a doubt,
Although the `title` has been changed "Welcome to the CovidNearMe App" to "Welcome to CovidNearMe". I left the semantics label of `title` unchanged to "Welcome to the Covid Near Me App".
I thought it would be more appropriate sentence for screen reader to read.
Is it fine or should i update semantic label for consistency?

#### Changes (All changes refers to Intro screen)
* Title changed from "Welcome to the CovidNearMe App" to "Welcome to CovidNearMe".
* Added a small description below the title.
* Button text changed from "LEARN MORE" to "GET STARTED".

#### Other changes due to above changes
* Updated app localizations.
* Updated tutorial_test, image_test, smoke_workflow_test and goldens.

### CODE REVIEW

**Comment by aadarsh-patel** [comment]
I am trying to regenerate the internationalization code using command,
`dart tools/gen_l10n.dart --arb-dir lib/src/l10n`
But i am getting following error,
```fatal: Unexpected character
#0      _ChunkedJsonParser.fail (dart:convert-patch/convert_patch.dart:1394:5)
#1      _ChunkedJsonParser.parse (dart:convert-patch/convert_patch.dart:847:56)
#2      _parseJson (dart:convert-patch/convert_patch.dart:31:10)
#3      JsonDecoder.convert (dart:convert/json.dart:495:36)
#4      JsonCodec.decode (dart:convert/json.dart:153:41)
#5      new AppResourceBundle (file:///C:/Users/Aadarsh/Desktop/tesla/coronavirus-diary/tools/gen_l10n_types.dart:369:14)
#6      LocalizationsGenerator.loadResources (file:///C:/Users/Aadarsh/Desktop/tesla/coronavirus-diary/tools/gen_l10n_lib.dart:534:46)
#7      main (file:///C:/Users/Aadarsh/Desktop/tesla/coronavirus-diary/tools/gen_l10n.dart:108:9)
<asynchronous suspension>
#8      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:305:32)
#9      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:174:12)
```

How can i solve this?

Edit: Issue has been resolved, was missing a comma in `app_en.erb`

**Comment by joshua-s** [comment]
@aadarsh-patel it was a good decision to leave the semantics as "Covid Near Me". Screen readers mispronounce "CovidNearMe"

File: lib/src/l10n/app_en.arb [hunk 1]
```diff
@@ -286,11 +286,16 @@
     "description": "Step 3 for the screen that lists 3 steps for staying safe"
   },
 
-  "tutorialIntroStepWelcome": "Welcome to the CovidNearMe App",
+  "tutorialIntroStepWelcome": "Welcome to CovidNearMe",
   "@tutorialIntroStepWelcome": {
     "description": "Heading on the initial screen. The name of the app, CovidNearMe, should not be translated"
   },
 
+  "tutorialIntroStepDescription": "CovidNearMe is a US Centers for Disease Control project to help the public respond to the novel coronavirus (COVID-19). It allows you to:",
```
Comment (ferndot) [inline lib/src/l10n/app_en.arb]: We don't have a current affiliation with the CDC. This is a potential alternative:
```suggestion
  "tutorialIntroStepDescription": "CovidNearMe lets you view statistics about COVID-19 in your local area and submit your symptoms to inform the disease response efforts.",
```

File: lib/src/l10n/app_localizations_en.dart [hunk 2]
```diff
@@ -220,7 +220,11 @@ If this is all OK and you are 18 years old or older, please click \"I agree\". O
       'Check back in if you continue to experience symptoms.';
 
   @override
-  String get tutorialIntroStepWelcome => 'Welcome to the CovidNearMe App';
+  String get tutorialIntroStepWelcome => 'Welcome to CovidNearMe';
+
+  @override
+  String get tutorialIntroStepDescription =>
+      'CovidNearMe is a US Centers for Disease Control project to help the public respond to the novel coronavirus (COVID-19). It allows you to:';
```
Comment (ferndot) [inline lib/src/l10n/app_localizations_en.dart]: Ditto from above.

**Review by ferndot — CHANGES_REQUESTED** [review]
Hey, thanks for working on this. It looks good, but needs a change to the description string.

**Comment by aadarsh-patel** [comment]
Thanks for the review.
Happy to contribute!

**Comment by ferndot** [comment]
Merged, thanks @aadarsh-patel!

**Pull request merged**